proparse/src/main/java/org/prorefactor/core/JsonNodeLister.java

```diff
@@ -34,7 +40,11 @@ public JsonNodeLister(JPNode topNode, PrintWriter writer, Integer[] ignoredKeywo
    * Print node content to PrintWriter
    */
   public void print() {
-    printSub(topNode, true);
+    try {
+      printSub(topNode, true);
+    } catch (IOException uncaught) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1166)